pep-9999.rst

```diff
@@ -0,0 +1,675 @@
+PEP: 9999
+Title: Literal types
+Author: Some subset of the mypy team
```

Comment:
Maybe some people from Pyre, pytype also want to be here?

Reply:
Sure, I guess -- I have no idea how authorship works on PEPs, to be honest.